oap-server/server-core/src/test/resources/component-libraries.yml

```diff
@@ -335,6 +335,9 @@ AspNet:
 SmartSql:
   id: 3016
   languages: C#
+FreeSql:
+  id: 3017
+  languages: C#
```

Comment:
You don't need to update this file. This is a file for testing only

Reply:
Thanks ,i will remove the content